Gemfile

```diff
@@ -28,4 +33,4 @@ group :test do
   gem 'launchy'
 end
 
-gem 'rails_12factor', group: :production
\ No newline at end of file
+gem 'rails_12factor', group: :production
```

Comment:
Nie jestem miłośnikiem gemów "production-only" - nawet jeśli jest to tylko 12factor. Jeśli zmienia zachowanie aplikacji w produkcji, to powinien to również robić w developmencie żeby dało się przetestować. Jeśli ten gem faktycznie nie działa z developmentem to warto taką informację zawrzeć w komentarzu i dalej sortować gemy po nazwie.